lib/dradis/plugins/content_service/nodes.rb

```diff
@@ -74,7 +74,7 @@ def nodes_from_evidence
     #
     # Returns and Array with a unique collection of Nodes.
     def nodes_from_properties
-      Node.user_nodes.where('properties IS NOT NULL AND properties != \'{}\'')
+      Node.user_nodes.where("type_id <> #{Node::Types::CONTENTLIB} AND properties IS NOT NULL AND properties != '{}'")
```

Comment:
The `:user_nodes` scope already excludes `CONTENTLIB`

Reply:
I think `user_nodes` work, this change was an error. Reverted.